boards/arm/colibri_imx7d_m4/colibri_imx7d_m4.yaml

```diff
@@ -20,3 +20,4 @@ testing:
     - bluetooth
 supported:
   - pwm
+vendor: nxp
```

Comment:
while this has an NXP part on it it is not an NXP board from what I can tell. This board came from Tordex

Reply:
right, we see this for many boards. So I am not sure here if this means the board compatible in dts is wrong here and if it needs to be fixed.